docs/content/configuration.md

```diff
@@ -516,6 +516,42 @@ bundles:
       max_delay_seconds: 120
 ```
 
+#### Azure Managed Identities Token
```

Comment:
Great docs!

Reply:
Thanks 😄 I have no idea if the example makes any sense but I'll be amending that as necessary once I get an example manually working.